mcp/gcp_ae.tf

```diff
@@ -1,7 +1,7 @@
 
 //noinspection HILUnresolvedReference
 data "google_project" "self" {
-  count = lookup(local.gae, "create_google_project", false) ? 0 : 1
+  count = local.gae == {} ? 0 : lookup(local.gae, "create_google_project", false) ? 0 : 1
```

Comment:
Interesting. I wasn't aware that nested conditions had become a thing. Looks like you could have used logical OR as well `||`